clifford/__init__.py

```diff
@@ -117,7 +117,8 @@ def linear_operator_as_matrix(func, input_blades, output_blades):
     ndimout = len(output_blades)
     mat = np.zeros((ndimout, ndimin))
     for i, b in enumerate(input_blades):
-        mat[:, i] = np.array([func(b)[j] for j in output_blades])
```

Comment:
Nice find

Reply:
Probably makes sense to spin out a quick PR with just this fix against master.

Reply:
yeah I was thinking that